File: front/src/app/core/domain/message/answer-question.message.ts
import {GamePlayerResultDto} from "@domain/games/game-player-result.dto";
import {MessageTypeEnum} from "@domain/message/message-type.enum";
import { BaseMessage } from "./base.message";

export class AnswerQuestionMessage extends BaseMessage {
    override readonly type = MessageTypeEnum.ANSWER_QUESTION;
    response: GamePlayerResultDto;
}

File: back/src/main/java/fr/uha/ensilan/concours/back/dto/message/AnswerQuestionMessage.java
package fr.uha.ensilan.concours.back.dto.message;

import fr.uha.ensilan.concours.back.dto.game.GamePlayerResultDto;
import lombok.Getter;
import lombok.Setter;


@Getter
@Setter
public class AnswerQuestionMessage extends BaseMessage {
    private GamePlayerResultDto response;

    public AnswerQuestionMessage() {
        super(MessageType.ANSWER_QUESTION);
    }
}
